WebHostLib/static/styles/tracker__Celeste64.css

```diff
@@ -0,0 +1,142 @@
+@import url('https://fonts.googleapis.com/css2?family=Lexend+Deca:wght@100..900&display=swap');
+
+.tracker-container {
+    width: 440px;
+    box-sizing: border-box;
+    font-family: "Lexend Deca", Arial, Helvetica, sans-serif;
+    border: 2px solid black;
+    border-radius: 4px;
+    resize: both;
+
+    background-color: #000000;
+    color: white;
+}
+
+.hidden {
+    display: none !important;
```

Comment:
Please avoid using `!important` unless there is absolutely no alternative. Instead, try using a more specific rule.